web/frontend/src/components/slam.vue

```diff
@@ -37,8 +41,22 @@ const viewSLAMPCDMap = (name: string) => {
       if (error) {
         return displayError(error);
       }
-      const pcObject = response!.getPointCloud()!;
-      pointcloud = pcObject.getPointCloud_asU8();
+      pointcloud = Uint8Array.from(atob(response!.toObject().pointCloud?.pointCloud), c => c.charCodeAt(0));
```

Comment:
This is what fixed the `three.module.js:10155 THREE.BufferGeometry.computeBoundingSphere(): Computed radius is NaN. The "position" attribute is likely to have NaN values.` exception within the PCD loader. 

This means that 
1. `response!.getPointCloud()!.getPointCloud_asU8()` is returning different data than
2. `Uint8Array.from(atob(response!.toObject().pointCloud?.pointCloud), c => c.charCodeAt(0))` 

the first version is adding these 26 bytes at the front of the byte array:
`[26, 14, 112, 111, 105, 110, 116, 99, 108, 111, 117, 100, 47, 112, 99, 100, 10,
 209, 211, 166, 1, 10, 204, 211, 166, 1]`

These bytes ` 112, 111, 105, 110, 116, 99, 108, 111, 117, 100, 47, 112, 99, 100, 10`

are the following string in ascii:
`pointcloud/pcd\n`

which is what is written to 
https://github.com/viamrobotics/api/blob/main/proto/viam/service/slam/v1/slam.proto#L115
GetMapResponse.mime_type

As a result, I think `response!.getPointCloud()!.getPointCloud_asU8()` might be returning the full `GetMapResponse` proto response, not the content of `GetMapResponse.point_cloud.point_cloud` which is what we want.